app/views/layouts/component.njk

```diff
@@ -33,10 +33,12 @@
   {% endblock %}
   </p>
 
+  {% if componentGuidanceLink %}
   <h2 class="govuk-heading-l">Guidance</h2>
   <p class="govuk-body">
-    More information about when to use {{ componentName }} can be found on <a href="{{- componentGuidanceLink -}}" title="Link to read guidance on the use of {{ componentName }} on Gov.uk Design system website" class="govuk-link">GOV.UK Design System</a>
+    More information about when to use {{ componentName }} can be found on <a href="{{- componentGuidanceLink -}}" title="Guidance on the use of {{ componentName | replace("-", " ") | capitalize }} on GOV.UK Design System" class="govuk-link">GOV.UK Design System</a>.
```

Comment:
Should we be using title here? It doesn't improve accessibility as far as I understood?

https://silktide.com/i-thought-title-text-improved-accessibility-i-was-wrong/